webasto/integrationpluginwebasto.cpp

```diff
@@ -284,19 +642,43 @@ void IntegrationPluginWebasto::evaluatePhaseCount(Thing *thing)
         if (amperePhase3 != 0)
             phaseCount += 1;
 
-        thing->setStateValue(liveWallboxPhaseCountStateTypeId, phaseCount);
+        thing->setStateValue(webastoLivePhaseCountStateTypeId, phaseCount);
+    }
+}
+
+void IntegrationPluginWebasto::executeWebastoNextPowerAction(ThingActionInfo *info, bool power)
+{
+    qCDebug(dcWebasto()) << (power ? "Enable": "Disable") << "charging on" << info->thing();
```

Comment:
usual debug print nitpick: "Enabling/Disabling..." or it seems to instruct the user to go to the wallbox and do something manually.